specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ServerAzureADAdministrators.json

```diff
@@ -0,0 +1,416 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators/{administratorName}": {
+      "get": {
+        "tags": [
+          "ServerAzureADAdministrators"
+        ],
+        "description": "Gets a server Administrator.",
+        "operationId": "ServerAzureADAdministrators_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "administratorName",
+            "in": "path",
+            "description": "The name of server active directory administrator.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "ActiveDirectory"
+            ],
+            "x-ms-enum": {
+              "name": "AdministratorName",
+              "modelAsString": true
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a specified Azure Active Directory administrator.",
+            "schema": {
+              "$ref": "#/definitions/ServerAzureADAdministrator"
+            }
+          },
+          "default": {
```

Comment:
The "default" response should have a schema consistent with the [common error contract](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content). 
Here is an example:
https://github.com/Azure/azure-rest-api-specs/blob/0cf70260108cb40bc68d408674c187ed065f8c64/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L105

Reply:
Let's file this as an issue for SQL RP in general because we will need to implement it in our generator.